src/Traits/AssertionsTrait.php

```diff
@@ -85,6 +85,27 @@ public function assertJsonDataCount(int $count, string $url, array $headers = []
         return $response;
     }
 
+    /**
+     * @param string $url
+     * @param array  $headers
+     * @param array  $ids
+     */
+    private function assertJsonDataContainsIds(string $url, array $ids = [], array $headers = [])
+    {
+        $response = $this->json('GET', $url, [], $headers);
+        $content = json_decode($response->getContent());
+
+        $this->assertTrue(property_exists($content, 'data'), print_r($content, true));
+
+        $this->assertCount(count($ids), $content->data);
+        $expectedIds = array_map(function ($item) {
+            return $item->id;
+        }, $content->data);
+        $this->assertEquals($ids,
```

Comment:
When arguments are split across multiple lines, the first argument should be on a new line and there should only be one argument per line: https://github.com/php-fig/fig-standards/blob/master/proposed/extended-coding-style-guide.md#45-method-and-function-arguments

```
Argument lists MAY be split across multiple lines, where each subsequent line is indented once. When doing so, the first item in the list MUST be on the next line, and there MUST be only one argument per line.

When the argument list is split across multiple lines, the closing parenthesis and opening brace MUST be placed together on their own line with one space between them.
```